cfgov/jinja2/v1/_includes/related-content.html

```diff
@@ -24,7 +24,7 @@
 <div class="related-links">
     <h2 class="header-slug">
         <span class="header-slug_inner">
-            Related links
+            Related content
```

Comment:
Yes, I understand. I'm referring _only_ to this file. The file above, the one in `molecules/`, is the one that needs to be updated for the Wagtail module. _This_ file seems to only be called directly by some templates, like Events and Newsroom.